Content/GUI/Tree.cs

```diff
@@ -52,8 +57,8 @@ public override void Draw(SpriteBatch spriteBatch)
 				var inner = new InnerPanel();
 				inner.Left.Set(0, 0);
 				inner.Top.Set(0, 0);
-				inner.Width.Set(600, 0);
-				inner.Height.Set(800, 0);
+				inner.Width.Set(PanelWidth - 0, 0);

```

Comment:
minus zero seems extraneous here?

Reply:
Ahh yeh I had a variable there and removed it haha. Will get this cleaned up